app/helpers/mustache_helper.rb

```diff
@@ -448,4 +444,50 @@ def news_item_img_src(item)
     mo = MediaObject.find_by_source_url_hash(MediaObject.hash_source_url(url))
     mo.nil? ? nil : mo.file.url(:medium)
   end
+
+  def hero_config(hero_image)
```

Comment:
Assignment Branch Condition size for hero_config is too high. [17.15/15]<br>Method has too many lines. [12/10]